mm/2s2h/BenGui/BenGui.cpp

```diff
@@ -164,9 +165,8 @@ void SetupGuiElements() {
         "gWindows.CheckTrackerSettings", "Check Tracker Settings");
     gui->AddGuiWindow(mRandoCheckTrackerSettingsWindow);
 
-    mModalWindow = std::make_shared<BenModalWindow>("gWindows.ModalWindow", "Modal Window");
-    gui->AddGuiWindow(mModalWindow);
-    mModalWindow->Show();
+    mQuickStartMenu = std::make_shared<Ship::QuickStart>("gWindows.QuickStart.", "QuickStart Menu");
```

Comment:
The trailing period results in a blank attribute under the QuickStart config section.
```suggestion
    mQuickStartMenu = std::make_shared<Ship::QuickStart>("gWindows.QuickStart", "QuickStart Menu");
```